### DISCUSSION TITLE
Add permission for modifying light blocks

### DISCUSSION THREAD

**Discussion by ToeDev** [PaperMC/Paper#10581] [Ideas]
### Is your feature request related to a problem?

In 1.19.3, minecraft added the minecraft:light block to operator only blocks.

### Describe the solution you'd like.

If possible, it would be ideal to have light block modification via right clicking a permission node instead of op-only.  Many of my creative server players used to use light blocks, but now this is impossible.

### Describe alternatives you've considered.

I could make a small plugin to simulate the previous behavior, but this seems like something that would make more sense in a Paper permission node or config.

**Comment by Gydo194** [comment]
Can I work on this? I've never worked on PaperMC before, but I'd like to help out.

**Comment by lynxplay** [comment]
I think this is going to be handled by #9339

**Comment by Gydo194** [comment]
ah alright
